Jenkinsfile

```diff
@@ -510,7 +510,18 @@ try {
           init_git()
           unpack_lib('gpu')
           timeout(time: max_time, unit: 'MINUTES') {
-              sh "${docker_run} gpu ./perl-package/test.sh"
+            sh "${docker_run} gpu ./perl-package/test.sh"
+          }
+        }
+      }
+    },
+    'Cpp: GPU': {
+      node('mxnetlinux-gpu') {
+        ws('workspace/build-cmake-gpu') {
```

Comment:
Change workspace name

Reply:
Seems like it was copy&pasted from the build job

Reply:
Bah, somehow missed this (I think during a merge conflict).  Good catch.